scripts/protocgen2.sh

```diff
@@ -0,0 +1,30 @@
+ # this script is for generating protobuf files for the new google.golang.org/protobuf API
```

Comment:
not a fan of these scripts actually. 

 Ignite CLI automatically gets the proto files and uses the same versions in your go mod, scripts always use latest versions which is not ideal

Reply:
Version of `cosmos-proto`?

Reply:
The official way to track tool dependencies in a module (not sure this applies well in the execution context of this script) :
https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module

Also mentioned here : https://github.com/ignite/cli/pull/2680#discussion_r933353893

Reply:
> Version of cosmos-proto?


Not sure if I understand what do you mean.

Users shouldn't need to maintain a script file to build code from proto. It's not the most pleasant way of doin it and also makes it hard to keep track of versions. CLI on the other hand checks your go.mod and retrieves the correct versions for the proto files that your chain depends on for code generation, instead of relying on "latest"

Reply:
We are not imposing anything, `spn` users can use the already built proto

For Ignite in general, I think we should let the freedom to the user to choose the tool they want to use, either Ignite by default or `buf` like many projects in the ecosystem do like `cosmos-sdk`

Reply:
Let's move forward?

We need the proto formatting tool in any case, we can revert for proto generation if there is issues with this solution in the future.

We can also open a specific issue if there is a strong opinion about only relying on Ignite for proto